Pet-Clinic-Frontend/README.md

```diff
@@ -28,3 +28,15 @@ export default {
 - Replace `plugin:@typescript-eslint/recommended` to `plugin:@typescript-eslint/recommended-type-checked` or `plugin:@typescript-eslint/strict-type-checked`
 - Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
 - Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+
+## How to start project
+
+in terminal
+
+- cd my-project
+- npm install
+- npm run dev
```

Comment:
Can we use the 

(triple backticks)bash notation here rather? It will make it copyable.